pkg/kubelet/util/pluginwatcher/example_plugin_apis/v1beta2/api.proto

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This file was added in 2018: https://github.com/kubernetes/kubernetes/pull/63328.